app/src/main/java/com/cyberbot/checkers/game/AiPlayer.java

```diff
@@ -89,6 +93,41 @@ private int minmax(Node node, int depth, boolean maximizingPlayer) {
         }
     }
 
+    private int alphabeta(Node node, int depth, int lvl2, int alpha, int beta, boolean maximizingPlayer) {
```

Comment:
In order to comply with standard names please capitalize the word "beta".

Reply:
OK.